data/schemas/keyboard.jsonschema

```diff
@@ -293,7 +293,8 @@
                             "enum": ["custom", "embedded_flash", "legacy", "rp2040_flash", "spi_flash"]
                         },
                         "backing_size": {"$ref": "qmk.definitions.v1#/unsigned_int"},
-                        "logical_size": {"$ref": "qmk.definitions.v1#/unsigned_int"}
+                        "logical_size": {"$ref": "qmk.definitions.v1#/unsigned_int"},
+                        "protect_last_sectors": {"$ref": "qmk.definitions.v1#/unsigned_int"}
```

Comment:
maybe `WEAR_LEVELING_EFL_FIRST_SECTOR`, `WEAR_LEVELING_EFL_FLASH_SIZE` should also be here?